pkg/sql/parser/sql.y

```diff
@@ -3839,7 +3839,7 @@ alter_external_connection_stmt:
 				 ConnectionLabelSpec: *($4.labelSpec()),
 		     As: $6.expr(),
 		}
-	} 
+	}
```

Comment:
Something wonky must be going on with my formatter, no way I'm the only one whose formatter will clean up these trailing spaces.